site/cypress/e2e/polls.spec.js

```diff
@@ -279,6 +283,62 @@ describe('Test cases revolving around polls functionality', () => {
         cy.get('.radio').eq(3).contains('Answer 2');
         cy.get('.radio > input').eq(3).should('not.be.checked');
 
+        // log into instructor and edit histogram, answer release and close the poll
+        cy.logout();
+        cy.login();
+        cy.visit(['sample', 'polls']);
+        cy.contains('Poll Cypress Test').siblings(':nth-child(6)').children().click();
+        cy.contains('Poll Cypress Test').siblings(':nth-child(1)').children().click();
+        cy.url().should('include', 'sample/polls/editPoll');
+        cy.get('#breadcrumbs > :nth-child(7) > span').should('have.text', 'Edit Poll');
+        cy.get('#student-histogram-release-setting').select('when_ended');
+        cy.get('#student-answer-release-setting').select('when_ended');
+        cy.get('button[type=submit]').click();
+        cy.wait(1000);
+
+        // log into student to verify histogram and answer are released
+        cy.logout();
+        cy.login('student');
+        cy.visit(['sample', 'polls']);
+        cy.contains('Poll Cypress Test').siblings(':nth-child(3)').contains('View Poll').click();
+        cy.get('#toggle-histogram-button').click();
+        cy.wait(1000);
```

Comment:
As per the comment:
> log into student to verify histogram and answer are released

I think some additional logic needs to happen here to actually check that the histogram and answer are properly released.  Perhaps you can target something on the histogram, such as checking the color of each bar.

The same comment applies throughout the rest of these changes.